core/src/main/java/io/questdb/cairo/TableUtils.java

```diff
@@ -194,6 +202,7 @@ public final class TableUtils {
     public static final long TX_OFFSET_LAG_ROW_COUNT_32 = TX_OFFSET_LAG_TXN_COUNT_32 + 4;
     public static final long TX_OFFSET_LAG_MIN_TIMESTAMP_64 = TX_OFFSET_LAG_ROW_COUNT_32 + 4;
     public static final long TX_OFFSET_LAG_MAX_TIMESTAMP_64 = TX_OFFSET_LAG_MIN_TIMESTAMP_64 + 8;
+    public static final long TX_OFFSET_MAT_VIEW_BASE_TXN_64 = TX_OFFSET_LAG_MAX_TIMESTAMP_64 + 8;
```

Comment:
not used anymore

Reply:
Removed in 89e3665b3d9d55d9e442abc04cca05ab12915062